### PR TITLE
Fix some typos (found by codespell)

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: src/lib/openjp2/j2k.c [hunk 1]
```diff
@@ -8829,7 +8829,7 @@ static OPJ_BOOL opj_j2k_read_header_procedure(opj_j2k_t *p_j2k,
         if (l_marker_handler->id == J2K_MS_UNK) {
             if (! opj_j2k_read_unk(p_j2k, p_stream, &l_current_marker, p_manager)) {
                 opj_event_msg(p_manager, EVT_ERROR,
-                              "Unknow marker have been detected and generated error.\n");
+                              "Unknown marker have been detected and generated error.\n");
```
Comment (stweil) [inline src/lib/openjp2/j2k.c:8832]: This looks also like a singular / plural mismatch. Should it be `markers have` or `marker has`?

Comment (rouault) [reply src/lib/openjp2/j2k.c:8832]: should be singular

**Comment by rouault** [comment]
the astyle check fails because you've touched files that weren't compliant with it before. Would you mind running astyle on them first to have a reindent-only commit, and then fix the typos in a separate commit ?

**Comment by stweil** [comment]
Would it be also okay to add the astyle commit after the typo fixes (which required some manual work)?

**Comment by rouault** [comment]
> Would it be also okay to add the astyle commit after the typo fixes (which required some manual work)?

yeah, that can do it too

File: src/lib/openjp2/j2k.c (hunk 1 above)
Comment (stweil) [reply src/lib/openjp2/j2k.c]: Fixed in 2nd commit.

**Comment by stweil** [comment]
Only a single source file had to be re-formatted (done in 3rd commit).

File: src/lib/openjp2/j2k.c [hunk 2]
```diff
@@ -50,7 +50,7 @@
 /*@{*/
 
 /**
- * Sets up the procedures to do on reading header. Developpers wanting to extend the library can add their own reading procedures.
+ * Sets up the procedures to do on reading header. Developers wanting to extend the library can add their own reading procedures.
```
Comment (stweil) [inline src/lib/openjp2/j2k.c:53]: @rouault, you could use the same change in your GitHub profile.

Comment (rouault) [reply src/lib/openjp2/j2k.c:53]: thanks :-) Frenchism corrected

**Pull request merged**